File: Assignment-7/Backend/Models/Vehicle.cs
namespace LogisticApp.Models
{
    public class Vehicle
    {
        public int VehicleId { get; set; }
        public string VehicleNumber { get; set; }
        public string Type { get; set; }
        public int Capacity { get; set; } // Changed from int to string
        public bool IsAvailable { get; set; } = true;
    }
}

File: Assignment-8/Frontened/src/app/models/vehicle.ts
// src/app/models/vehicle.ts
export interface Vehicle {
  vehicleId: number;          // Unique ID from backend
  vehicleNumber: string;      // Example: MH12AB1234
  type: string;               // Example: Truck, Van
  capacity: number;           // Capacity in tons or units
  isAvailable: boolean;       // True = Available, False = In Use
  availability?: string;      // Optional display string like "Available" | "In Use"
}
